db/sqlite_test.go

```diff
@@ -0,0 +1,63 @@
+package db
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Civil/github2telegram/configs"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	testDbName = "testgithub2telegram.dbdata"
+)
+
+type SQLiteSuite struct {
+	suite.Suite
+	db Database
+}
+
+func (s *SQLiteSuite) SetupSuite() {
+	configs.Config.DatabaseURL = fmt.Sprintf("./%s", testDbName)
```

Comment:
I'd rather use something in tmpdir, e.x. https://golang.org/pkg/io/ioutil/#TempDir and use current timestamp in a file name.

Reply:
I will check it and rework. TempDir looks better, i agree.

Reply:
Fixed